### DISCUSSION TITLE
Request for Allo Boss2 drivers

### DISCUSSION THREAD

**Discussion by Shiningpath34** [rern/rAudio#847] [Feature Requests]
Hello team,

rAudio is rocking on my Raspberrypi 4 (setup and config hardly took 10min). For me this is a dream software with all the features one could wish for!! Thank you for the great software and keep up the good work. If there is option for donation, please let me know.
My rpi has a Allo Boss2 hat and I couldnt find the driver in i2c settings. Is there anyway you could add this?

**Comment by rern** [comment]
Done.

**Reply by Shiningpath34** [reply]
Thank you for the fast response!! Much appreciated...
